jcontroller.php

```diff
@@ -172,6 +177,25 @@ protected function refRequest($param)
 
         return $this->refRequest[$param];
     }
+    
+    /**
+     * Filters access to an action based on the controller methods.
+     *  
+     * If access is granted or is not mentioned, it proceeds to display the action.
+     * If access is not granted, it displays an error code (401 - Access Forbidden) and exits.
+     * 
+     * @return void
+     */
+    public function filterAccessControl()
+    {
+        $actionName = $_GET['action'];
+        $accessMethod = $this->accessRules()[$actionName];
```

Comment:
We need the ability to pass in arguments to this access control methods

Reply:
Vaia could you please explain in where do you want to pass in the arguments?